docs/book/stacks-and-components/stack-deployment/cloud-stacks/minimal-gcp-stack.md

```diff
@@ -0,0 +1,179 @@
+<figure><img src="../../../.gitbook/assets/GCP_Service_Connector.png" alt=""><figcaption><p>Choose the GCP Connector</p></figcaption></figure>
+
+<figure><img src="../../../.gitbook/assets/GCP_Connector_Key.png" alt=""><figcaption><p>Paste the entire contents of the key.json here</p></figcaption></figure>
+
+<figure><img src="../../../.gitbook/assets/GCP_Connector_Resources.png" alt=""><figcaption><p>Make sure GCR, GCS and Generic GCP Resources are all selected here.</p></figcaption></figure>
+{% endtab %}
+{% endtabs %}
+
+### 6) Create Stack Components
+
+#### Artifact Store
+
+Before you run anything within the zenml CLI, head on over to GCP and create a GCS bucket, in case you don't have already one that you can use. Once this is done, you can create the zenml stack component as follows:
+
+{% tabs %}
+{% tab title="CLI" %}
+```bash
+export ARTIFACT_STORE_NAME=gcp_artifact_store
+
+# Register the GCS artifact-store and reference the target GCS bucket
+zenml artifact-store register ${ARTIFACT_STORE_NAME} --flavor gcp \
+    --path=gs://<YOUR_BUCKET_NAME>
+
+# Connect the GCS artifact-store to the target bucket via a GCP Service Connector
+zenml artifact-store connect ${ARTIFACT_STORE_NAME} -i
+```
+
+{% hint style="info" %}
+Head on over to our [docs](../../component-guide/artifact-stores/gcp/) to learn more about the registration of an artifact store.
+{% endhint %}
+{% endtab %}
+
+{% tab title="Dashboard" %}
+<figure><img src="../../../.gitbook/assets/Create_Artifact_Store.png" alt=""><figcaption><p>Choose the GCP Artifact Store.</p></figcaption></figure>
+
+<figure><img src="../../../.gitbook/assets/Register_Artifact_Store_Connector.png" alt=""><figcaption><p>Choose the name of your Artifact Store and the Connector you just created. This will allow you to pick the bucket of your choice.</p></figcaption></figure>
+{% endtab %}
+{% endtabs %}
+
+#### Orchestrator
+
+{% tabs %}
+{% tab title="CLI" %}
+```bash
+export ORCHESTRATOR_NAME=gcp_vertex_orchestrator
+
+# Register the GCS artifact-store and reference the target GCS bucket
+zenml orchestrator register ${ORCHESTRATOR_NAME} --flavor=vertex 
+  --project=<PROJECT_NAME> --location=europe-west2
+
+# Connect the GCS orchestrator to the target gcp project via a GCP Service Connector
+zenml orchestrator connect ${ORCHESTRATOR_NAME} -i
+```
+
+{% hint style="info" %}
+Head on over to our [docs](../../component-guide/orchestrators/vertex.md) to learn more about the registration of an orchestrator.
+{% endhint %}
+{% endtab %}
+
+{% tab title="Dashboard" %}
+<figure><img src="../../../.gitbook/assets/Create_Orchestrator.png" alt=""><figcaption><p>Select the Vertex Orchestrator</p></figcaption></figure>
+
+<figure><img src="../../../.gitbook/assets/Register_Orchestrator_Connector.png" alt=""><figcaption><p>Name it, Select the Connector and set an appropriate location. All other fields are optional.</p></figcaption></figure>
+{% endtab %}
+{% endtabs %}
+
+#### Container Registry
+
```

Comment:
do you need to tell them to create the registry as well?

if so, maybe add the gcloud CLI command as well?

Reply:
Not necessary, there is always a default one